_articles/finding-users.md

```diff
@@ -19,15 +19,15 @@ related:
 
 ## Spreading the word
 
-There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. If you are hoping others will find and use your open source project, however, it's time to tell everybody about your hard work!
+There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. Instead of hoping others will find and use your open source project, you have to _spread the word_ about your hard work!
 
 ## Figure out your message
 
-Before you start the actual work of promoting your project, you should be able to explain what it does, and why it matters.
+Before you start the actual work of promoting your project, you should be able to explain what your project does, and why your project matters.
 
-What makes your project different or interesting? Why did you create it? Answering these questions for yourself will make it easier to convince others.
+What makes your project different or interesting? Why did you create it? Answering these questions for yourself will help you communicate your project's significance.
 
-Remember that people get involved as users, and eventually contributors, because it solves a problem for them. As you think about your project's message and value, try to view it through the lens of what _they_ might want.
+Remember that people get involved as users, and eventually become contributors, because your project solves a problem for them. As you think about your project's message and value, try to view it through the lens of what _users_ might want.
 
```

Comment:
I feel like "they" is appropriate in this instance because it isn't just "users" but a collective of "users" and "contributors".

Reply:
Will change back to "they"

Reply:
I see thanks! 

In my above comment, I noticed a grammatical error ("try to view it" should be "try to view them"). After changing "it" to "them", I think that "they" should be replaced with "users and contributors". If we use "they", it is unclear if "they" is referring to the project's message and value or the users and contributors.